po-pgbouncer/version.txt

```diff
@@ -0,0 +1 @@
+master-22
```

Comment:
@alex-astronomer @vishwas-astro - should this be tag 22 or tag master-22
```
curl -X GET --header ‘Accept: application/json’ ‘https://registry.opensource.zalan.do/v2/acid/pgbouncer/tags/list’ | jq .
```

Reply:
I would retain the versioning of the upstream image as much as possible. so yeah `master-22`